app/forms/nsm/steps/letters_calls_form.rb

```diff
@@ -10,7 +10,11 @@ class LettersCallsForm < ::Steps::BaseFormObject
       attribute :calls_uplift, :integer
 
       validates :letters, numericality: { only_integer: true, greater_than_or_equal_to: 0, allow_blank: true }
+      validates :letters, numericality: { only_integer: true, greater_than_or_equal_to: 1, allow_blank: false },
```

Comment:
We're going to need some input from Chris on the wording of the error messages - we'll need full sentences explaining that you can't apply an uplift if no value is entered rather than 'must be greater than or equal to 1' which is likely to be confusing.

Also, to be fair, arguably the highlighted error field should be the 'Apply an uplift' checkbox rather than the number input? It might be worth getting Mary's eyes on this too as to what the intended behaviour is. Looking at the 'Expected behaviour' field on Jira, I think we might be a little bit early for code on this ticket.

Reply:
yeah, true! probably needs refinement on those two counts

Reply:
I've asked a question on Jira about this - we'll see what Mary and Chris say